libkbfs/md_journal.go

```diff
@@ -1178,7 +1226,11 @@ func (j *mdJournal) clear(
 	// Garbage-collect the old branch entries.  TODO: we'll eventually
 	// need a sweeper to clean up entries left behind if we crash
 	// here.
+	isLocalSquash := bid == LocalSquashBranchID
```

Comment:
Can you also add comments to `clear()`, now that it can leave entries behind? And some test coverage for the LocalSquashBranchID case.